src/Standards/PSR12/Docs/ControlStructures/BooleanOperatorPlacementStandard.xml

```diff
@@ -0,0 +1,57 @@
+<documentation title="Boolean Operator Placement">
+    <standard>
+    <![CDATA[
+    Boolean operators between conditions MUST always be at the beginning or at the end of the line, not a mix of both.
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: Boolean operator between conditions at the beginning of the line.">
+        =<![CDATA[
```

Comment:
> a well-formedness check would not flag the above typo

Technically there's nothing wrong from an XML perspective with that string. The `<![CDATA[` and `]]>` sequences just wrap a string value to avoid the inner text from being interpreted as XML, so they can be used anywhere a string is expected, or the string can be used without the wrapper. This is something that also wouldn't show up in a XSD validation check either, as a string is a string is a string. One can think of it like `<?= 'name &amp; description' ?>` versus `<?= htmlentities('name & description') ?>`; they both produce the same output.

Reply:
Oh wow I don't know how this got in 😬 🙈

Reply:
@dingo-d Happens to the best of us ;-)

@fredden That makes total sense to me. Thanks for adding that context.